migrations/Migration20240927110614.ts

```diff
@@ -0,0 +1,21 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20240927110614 extends Migration {
+    async up(): Promise<void> {
+        this.addSql(
+            'create table "user_lock" ("id" uuid not null, "created_at" timestamptz not null, "updated_at" timestamptz not null, "person_id" uuid not null, "locked_by" varchar(255) null, "locked_until" timestamptz null, constraint "user_lock_pkey" primary key ("id"));',
+        );
+
+        this.addSql(
+            'alter table "user_lock" add constraint "user_lock_person_id_foreign" foreign key ("person_id") references "person" ("id") on update cascade;',
+        );
+
+        this.addSql('alter table "service_provider" add column "requires2fa" boolean not null;');
```

Comment:
See above. Running this migration on an already deployed DB (that has already run the other migration) might crash because the 2fa-column already exists